docs/readmes/proposals/p004_apn_correction.md

```diff
@@ -0,0 +1,91 @@
+---
+id: p004_apn_correction
+title: MME APN Correction 
+hide_title: true
+---
+
+# Overview
+
+*Status: Accepted*\
+*Author: @ymasmoudi*\
+*Last Updated: 09/22*
+
+
+During session establishment, the phone may have an APN configured already.
+In this case, this APN will be used as it has a higher precedence over the
+one provided by the SIM card.
+
+This behavior requires a feature to override this value at the MME level
+based on IMSI prefixes prior to interrogating the HSS and creating the PDN
+session.
+
+This document describes the support of APN correction.
```

Comment:
The feature description is not clear to me. There is also repetition. 

Make it clear that this feature is for a federated set up where HSS is used and not used in standalone case.

Reply:
The description can be more specific:
" This feature enables the overriding of UE requested APN with a network specified APN via IMSI prefix based filtering. Up to 10 IMSI prefix filters and corresponding APNs to overwrite with can be defined."